### PR TITLE
Add proseminars requirement

### PR DESCRIPTION
**Pull request by JumiDeluxe**
Added rule for proseminars for both degrees.
Calculated:
At least 1 ECTS from seminar + proseminar, as seminars can also pass the requirement (afaik)

image

### CODE REVIEW

**Comment by jaemic00** [comment]
cool addition

**Comment by Ph0enixKM** [comment]
Facts, this should have been by now

File: planer-uwr-webapp/imports/ui/pages/PlanPage/RuleEntry.tsx [hunk 1]
```diff
@@ -32,6 +32,15 @@ const RuleEntry = (props: RuleEntryProps) => {
...
             );
+            case 'Proseminarium':
+                return (
+                    <span>
+                        <CourseTypeTag courseType={courseTypeById['14']} />
+                        <Separator value="+" />
+                        <CourseTypeTag courseType={courseTypeById['41']} />
+                        <Separator value="" />
+                    </span>
+                );
```
Comment (Florian3k) [inline planer-uwr-webapp/imports/ui/pages/PlanPage/RuleEntry.tsx:43]: Thanks for this addition. Could you just fix indentation in this place so it matches the code around?

Comment (JumiDeluxe) [reply planer-uwr-webapp/imports/ui/pages/PlanPage/RuleEntry.tsx]: Fixed

**Review by Florian3k — APPROVED** [review]
LGTM

**Pull request merged**